src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,169 @@
+    static int taskIndex=0;
+
+    public static void main(String[] args) {
+
+        greetUser("hi");
+        startChatbot();
+        greetUser("bye");
+    }
+
+    private static void startChatbot() {
+        Scanner input = new Scanner(System.in);
+        String task = "";
+        while(true) {
+            try {
+                task = input.nextLine();
+                System.out.println("    ____________________________________");
+                if (task.equals("bye")) {
+                    break;
+                } else if (task.equals("list")) {
+                    listTasks();
+                } else if (task.startsWith("mark ")) {
+                    markTask(task);
+                } else if (task.startsWith("unmark ")) {
+                    unmarkTask(task);
+                } else if (task.startsWith("todo") || task.startsWith("deadline") || task.startsWith("event")) {
+                    addTask(task);
+                } else {
+                    throw new DukeException();
+                }
+                System.out.println("    ____________________________________");
+            } catch (DukeException e) {
+                System.out.println("    Sorry I do not know what that means!");
+            }
+
+        }
+    }
+
+    private static void greetUser(String greeting) {
+        if(greeting.equals("hi")) {
+            String logo = " ____        _\n"
+                    + "|  _ \\ _   _| | _____\n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+            System.out.println(logo);
+            System.out.println("    ____________________________________");
+            System.out.println("    Hello! I'm Duke");
+            System.out.println("    What can I do for you?\n");
+        }
+        else {
+            System.out.println("    ____________________________________");
+            System.out.println("    Bye. Hope to see you again soon!");
+            System.out.println("    ____________________________________");
+        }
+    }
+
+    public static void listTasks() {
+        System.out.println("    Here are the tasks in your list:");
+        for(int i=0;i< taskList.length;i++) {
+            if(taskList[i] == null) break;
+            System.out.println("    "+(i+1)+": "+taskList[i]);
+        }
+    }
+
+    public static void markTask(String task) {
+        try {
+            task = task.replace("mark ", "");
+            int i = Integer.parseInt(task) - 1;
+            if (taskList[i] != null) {
+                taskList[i].setDone(true);
+                System.out.println("    Nice! I've marked this task as done:");
+                System.out.println("    " + taskList[i]);
+            } else {
+                System.out.println("    Please enter a valid task number");
+            }
+        } catch (IndexOutOfBoundsException e) {
+            if(taskIndex==0) {
```

Comment:
According to the Java coding standard, there should be white spaces between `if(` and around the equal signs.